### PR TITLE
Kevin Belford Lesson05

### PR DESCRIPTION
**Pull request by kcbelford**
Lesson05 Assignment

### CODE REVIEW

File: Lesson05/index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <script src="./script.js" charset="utf-8"></script>
+    <title>Lesson 05</title>
+  </head>
+  <body>
+    <h1>Bank of Belford</h1>
+    <button onclick="bankApp('Enter a letter to begin banking:\nQ - to quit application\nW - to withdraw funds\nD - to deposit funds\nB - to view balance')" type="button" name="button">Start Banking</button>
```
Comment (jimmynono) [inline Lesson05/index.html:10]: I'd rather this long string lives in your JS file, it's a bit long and clutters up the HTML.

File: Lesson05/script.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        bankApp(actions);
+      } while (input = 'W' || 'D' || 'B');
+      break;
+    case 'D':
+      let depositNum = prompt('Enter an amount to deposit:');
+      balance += Number(depositNum);
+
+      alert('Current balance: ' + '$' + balance);
+      do {
+        bankApp(actions);
```
Comment (jimmynono) [inline Lesson05/script.js:40]: recursion? Maybe a bit unneeded for this exercise

File: Lesson05/index.html (hunk 1 above)
Comment (kcbelford) [reply Lesson05/index.html:10]: Noted- I'll be sure to keep my html clutter-free

File: Lesson05/script.js (hunk 2 above)
Comment (kcbelford) [reply Lesson05/script.js:40]: I didn't even know that's what I was doing- haha!  I appreciated seeing your solution to this problem in class.  It was super helpful and I'm feeling better about how to structure my javascript more efficiently.

Comment (jimmynono) [reply Lesson05/script.js:40]: Yeah, a function that calls itself from inside. THE CALL IS COMING FROM INSIDE THE HOUSE!!!  :)

**Pull request closed without merging**